### DISCUSSION TITLE
yScale and curve type

### DISCUSSION THREAD

**Discussion by lednhatkhanh** [airbnb/visx#1851] [Q&A]
I want to display a tooltip on the maximum and minimum values of the line chart (I'm not using xychart). I tried to use yScale: `yScale(value)` and get the top from it. The problem is that it does not match with what renders on the screen, different curve type will render the chart differently thus making the top value incorrect. Could you please let me know do I calculate the top value of a point take the curve into consideration? Thank you.

**Comment by williaster** [comment]
Hi @lednhatkhanh it would help if you could add a visual explanation and code sandbox to your question! thanks.
